pro/src/screens/OfferEducational/OfferEducationalForm/FormDates/FormDates.tsx

```diff
@@ -19,11 +19,21 @@ const FormDates = ({
   disableForm,
   dateCreated,
 }: FormDatesProps): JSX.Element => {
-  const { values } = useFormikContext<OfferEducationalFormValues>()
+  const { values, setFieldValue } =
+    useFormikContext<OfferEducationalFormValues>()
   const minBeginningDate = dateCreated ? new Date(dateCreated) : new Date()
   const minDateForEndingDate = isDateValid(values.beginningDate)
     ? new Date(values.beginningDate)
     : new Date()
+
+  function beginningDateChangeHandler(e: ChangeEvent<HTMLInputElement>) {
```

Comment:
micro détail: privilégier les nom de fonctions qui commencent par des verbes d'action donc handleBlaBlaBla

Reply:
C'est noté merci!